tests/README.md

```diff
@@ -0,0 +1,197 @@
+# Test Strategy
```

Comment:
General Comment: 

I noticed we use both npm commands and pnpm commands in our READMEs now, it seems it's best to just use pnpm, I have this change ready locally and happy to push up some improvements @lukeocodes . Let me know if this is important to you.